contracts/Tokens/VAI/VAIController.sol

```diff
@@ -409,6 +448,16 @@ contract VAIController is VAIControllerStorageG2, VAIControllerErrorReporter, Ex
 
     // solhint-disable-next-line code-complexity
     function getMintableVAI(address minter) public view returns (uint, uint) {
+        if (mintEnabledOnlyForPrimeHolder && prime == address(0)) {
+            return (uint(Error.REJECTION), 0);
+        }
+
+        if (mintEnabledOnlyForPrimeHolder && prime != address(0)) {
+            if (!IPrime(prime).isUserPrimeHolder(minter)) {
```

Comment:
If you applied the previous suggested changes, we could simplify this check to this:
```
if (mintEnabledOnlyForPrimeHolder && !IPrime(prime).isUserPrimeHolder(minter)) {
  return (uint(Error.REJECTION), 0);
}
```